Update teamviewerqs

hwlans opened the pull request:
**Have you confirmed this pull request is not a duplicate?**
Yes

**Is this pull request creating or modifying a label in the fragments/labels folder, and not Installomator.sh itself?**
Yes

**Did you use [our editorconfig file](https://github.com/Installomator/Installomator/wiki/Contributing-to-Installomator)?**
Yes

**Additional context** Add any other context about the label or fix here.
Removed unnecessary `appName`

**Installomator log**
````
./assemble.sh teamviewerqs
2025-03-18 21:26:56 : INFO  : teamviewerqs : Total items in argumentsArray: 0
2025-03-18 21:26:56 : INFO  : teamviewerqs : argumentsArray:
2025-03-18 21:26:56 : REQ   : teamviewerqs : ################## Start Installomator v. 10.8beta, date 2025-03-18
2025-03-18 21:26:56 : INFO  : teamviewerqs : ################## Version: 10.8beta
2025-03-18 21:26:56 : INFO  : teamviewerqs : ################## Date: 2025-03-18
2025-03-18 21:26:56 : INFO  : teamviewerqs : ################## teamviewerqs
2025-03-18 21:26:56 : DEBUG : teamviewerqs : DEBUG mode 1 enabled.
2025-03-18 21:26:57 : INFO  : teamviewerqs : Reading arguments again:
2025-03-18 21:26:57 : DEBUG : teamviewerqs : name=TeamViewerQS
2025-03-18 21:26:57 : DEBUG : teamviewerqs : appName=
2025-03-18 21:26:57 : DEBUG : teamviewerqs : type=dmg
2025-03-18 21:26:57 : DEBUG : teamviewerqs : archiveName=
2025-03-18 21:26:57 : DEBUG : teamviewerqs : downloadURL=https://download.teamviewer.com/download/TeamViewerQS.dmg
2025-03-18 21:26:57 : DEBUG : teamviewerqs : curlOptions=
2025-03-18 21:26:57 : DEBUG : teamviewerqs : appNewVersion=15.63.4
2025-03-18 21:26:57 : DEBUG : teamviewerqs : appCustomVersion function: Not defined
2025-03-18 21:26:57 : DEBUG : teamviewerqs : versionKey=CFBundleShortVersionString
2025-03-18 21:26:57 : DEBUG : teamviewerqs : packageID=
2025-03-18 21:26:57 : DEBUG : teamviewerqs : pkgName=
2025-03-18 21:26:57 : DEBUG : teamviewerqs : choiceChangesXML=
2025-03-18 21:26:57 : DEBUG : teamviewerqs : expectedTeamID=H7UGFBUGV6
2025-03-18 21:26:57 : DEBUG : teamviewerqs : blockingProcesses=
2025-03-18 21:26:57 : DEBUG : teamviewerqs : installerTool=
2025-03-18 21:26:57 : DEBUG : teamviewerqs : CLIInstaller=
2025-03-18 21:26:57 : DEBUG : teamviewerqs : CLIArguments=
2025-03-18 21:26:57 : DEBUG : teamviewerqs : updateTool=
2025-03-18 21:26:57 : DEBUG : teamviewerqs : updateToolArguments=
2025-03-18 21:26:57 : DEBUG : teamviewerqs : updateToolRunAsCurrentUser=
2025-03-18 21:26:57 : INFO  : teamviewerqs : BLOCKING_PROCESS_ACTION=tell_user
2025-03-18 21:26:57 : INFO  : teamviewerqs : NOTIFY=success
2025-03-18 21:26:57 : INFO  : teamviewerqs : LOGGING=DEBUG
2025-03-18 21:26:57 : INFO  : teamviewerqs : LOGO=/System/Applications/App Store.app/Contents/Resources/AppIcon.icns
2025-03-18 21:26:57 : INFO  : teamviewerqs : Label type: dmg
2025-03-18 21:26:57 : INFO  : teamviewerqs : archiveName: TeamViewerQS.dmg
2025-03-18 21:26:57 : INFO  : teamviewerqs : no blocking processes defined, using TeamViewerQS as default
2025-03-18 21:26:57 : DEBUG : teamviewerqs : Changing directory to /Users/h.lans/Documents/GitHub/Installomator/build
2025-03-18 21:26:57 : INFO  : teamviewerqs : name: TeamViewerQS, appName: TeamViewerQS.app
2025-03-18 21:26:57 : WARN  : teamviewerqs : No previous app found
2025-03-18 21:26:57 : WARN  : teamviewerqs : could not find TeamViewerQS.app
2025-03-18 21:26:57 : INFO  : teamviewerqs : appversion:
2025-03-18 21:26:57 : INFO  : teamviewerqs : Latest version of TeamViewerQS is 15.63.4
2025-03-18 21:26:57 : REQ   : teamviewerqs : Downloading https://download.teamviewer.com/download/TeamViewerQS.dmg to TeamViewerQS.dmg
2025-03-18 21:26:57 : DEBUG : teamviewerqs : No Dialog connection, just download
2025-03-18 21:27:03 : INFO  : teamviewerqs : Downloaded TeamViewerQS.dmg – Type is  zlib compressed data – SHA is 256c3a0b5beae3eaff99781160914c9ae75e6b2e – Size is 60076 kB
2025-03-18 21:27:03 : DEBUG : teamviewerqs : DEBUG mode 1, not checking for blocking processes
2025-03-18 21:27:03 : REQ   : teamviewerqs : Installing TeamViewerQS
2025-03-18 21:27:03 : INFO  : teamviewerqs : Mounting /Users/h.lans/Documents/GitHub/Installomator/build/TeamViewerQS.dmg
2025-03-18 21:27:06 : DEBUG : teamviewerqs : Debugging enabled, dmgmount output was:
Checksumming Protective Master Boot Record (MBR : 0)…
Protective Master Boot Record (MBR :: verified CRC32 $BC149447
Checksumming GPT Header (Primary GPT Header : 1)…
GPT Header (Primary GPT Header : 1): verified CRC32 $0D853B92
Checksumming GPT Partition Data (Primary GPT Table : 2)…
GPT Partition Data (Primary GPT Tabl: verified CRC32 $F093731F
Checksumming  (Apple_Free : 3)…
(Apple_Free : 3): verified CRC32 $00000000
Checksumming disk image (Apple_HFS : 4)…
disk image (Apple_HFS : 4): verified CRC32 $4C7F6BEB
Checksumming  (Apple_Free : 5)…
(Apple_Free : 5): verified CRC32 $00000000
Checksumming GPT Partition Data (Backup GPT Table : 6)…
GPT Partition Data (Backup GPT Table: verified CRC32 $F093731F
Checksumming GPT Header (Backup GPT Header : 7)…
GPT Header (Backup GPT Header : 7): verified CRC32 $FC3D163A
verified CRC32 $BD6994C0
/dev/disk4          	GUID_partition_scheme
/dev/disk4s1        	Apple_HFS                      	/Volumes/TeamViewerQS

2025-03-18 21:27:06 : INFO  : teamviewerqs : Mounted: /Volumes/TeamViewerQS
2025-03-18 21:27:06 : INFO  : teamviewerqs : Verifying: /Volumes/TeamViewerQS/TeamViewerQS.app
2025-03-18 21:27:06 : DEBUG : teamviewerqs : App size: 173M	/Volumes/TeamViewerQS/TeamViewerQS.app
2025-03-18 21:27:08 : DEBUG : teamviewerqs : Debugging enabled, App Verification output was:
/Volumes/TeamViewerQS/TeamViewerQS.app: accepted
source=Notarized Developer ID
origin=Developer ID Application: TeamViewer Germany GmbH (H7UGFBUGV6)

2025-03-18 21:27:08 : INFO  : teamviewerqs : Team ID matching: H7UGFBUGV6 (expected: H7UGFBUGV6 )
2025-03-18 21:27:08 : INFO  : teamviewerqs : Installing TeamViewerQS version 15.63.4 on versionKey CFBundleShortVersionString.
2025-03-18 21:27:08 : INFO  : teamviewerqs : App has LSMinimumSystemVersion: 12.5.1
2025-03-18 21:27:08 : DEBUG : teamviewerqs : DEBUG mode 1 enabled, skipping remove, copy and chown steps
2025-03-18 21:27:08 : INFO  : teamviewerqs : Finishing...
2025-03-18 21:27:11 : INFO  : teamviewerqs : name: TeamViewerQS, appName: TeamViewerQS.app
2025-03-18 21:27:11 : WARN  : teamviewerqs : No previous app found
2025-03-18 21:27:11 : WARN  : teamviewerqs : could not find TeamViewerQS.app
2025-03-18 21:27:11 : REQ   : teamviewerqs : Installed TeamViewerQS, version 15.63.4
2025-03-18 21:27:11 : INFO  : teamviewerqs : notifying
2025-03-18 21:27:11 : DEBUG : teamviewerqs : Unmounting /Volumes/TeamViewerQS
2025-03-18 21:27:11 : DEBUG : teamviewerqs : Debugging enabled, Unmounting output was:
"disk4" ejected.
2025-03-18 21:27:11 : DEBUG : teamviewerqs : DEBUG mode 1, not reopening anything
2025-03-18 21:27:11 : REQ   : teamviewerqs : All done!
2025-03-18 21:27:11 : REQ   : teamviewerqs : ################## End Installomator, exit code 0
````

Please identify any issues fixed by your pull request by including the issue number. (Example: "Fixes #XXXX")

hwlans committed:
Update teamviewerqs.sh

````
./assemble.sh teamviewerqs
2025-03-18 21:26:56 : INFO  : teamviewerqs : Total items in argumentsArray: 0
2025-03-18 21:26:56 : INFO  : teamviewerqs : argumentsArray:
2025-03-18 21:26:56 : REQ   : teamviewerqs : ################## Start Installomator v. 10.8beta, date 2025-03-18
2025-03-18 21:26:56 : INFO  : teamviewerqs : ################## Version: 10.8beta
2025-03-18 21:26:56 : INFO  : teamviewerqs : ################## Date: 2025-03-18
2025-03-18 21:26:56 : INFO  : teamviewerqs : ################## teamviewerqs
2025-03-18 21:26:56 : DEBUG : teamviewerqs : DEBUG mode 1 enabled.
2025-03-18 21:26:57 : INFO  : teamviewerqs : Reading arguments again:
2025-03-18 21:26:57 : DEBUG : teamviewerqs : name=TeamViewerQS
2025-03-18 21:26:57 : DEBUG : teamviewerqs : appName=
2025-03-18 21:26:57 : DEBUG : teamviewerqs : type=dmg
2025-03-18 21:26:57 : DEBUG : teamviewerqs : archiveName=
2025-03-18 21:26:57 : DEBUG : teamviewerqs : downloadURL=https://download.teamviewer.com/download/TeamViewerQS.dmg
2025-03-18 21:26:57 : DEBUG : teamviewerqs : curlOptions=
2025-03-18 21:26:57 : DEBUG : teamviewerqs : appNewVersion=15.63.4
2025-03-18 21:26:57 : DEBUG : teamviewerqs : appCustomVersion function: Not defined
2025-03-18 21:26:57 : DEBUG : teamviewerqs : versionKey=CFBundleShortVersionString
2025-03-18 21:26:57 : DEBUG : teamviewerqs : packageID=
2025-03-18 21:26:57 : DEBUG : teamviewerqs : pkgName=
2025-03-18 21:26:57 : DEBUG : teamviewerqs : choiceChangesXML=
2025-03-18 21:26:57 : DEBUG : teamviewerqs : expectedTeamID=H7UGFBUGV6
2025-03-18 21:26:57 : DEBUG : teamviewerqs : blockingProcesses=
2025-03-18 21:26:57 : DEBUG : teamviewerqs : installerTool=
2025-03-18 21:26:57 : DEBUG : teamviewerqs : CLIInstaller=
2025-03-18 21:26:57 : DEBUG : teamviewerqs : CLIArguments=
2025-03-18 21:26:57 : DEBUG : teamviewerqs : updateTool=
2025-03-18 21:26:57 : DEBUG : teamviewerqs : updateToolArguments=
2025-03-18 21:26:57 : DEBUG : teamviewerqs : updateToolRunAsCurrentUser=
2025-03-18 21:26:57 : INFO  : teamviewerqs : BLOCKING_PROCESS_ACTION=tell_user
2025-03-18 21:26:57 : INFO  : teamviewerqs : NOTIFY=success
2025-03-18 21:26:57 : INFO  : teamviewerqs : LOGGING=DEBUG
2025-03-18 21:26:57 : INFO  : teamviewerqs : LOGO=/System/Applications/App Store.app/Contents/Resources/AppIcon.icns
2025-03-18 21:26:57 : INFO  : teamviewerqs : Label type: dmg
2025-03-18 21:26:57 : INFO  : teamviewerqs : archiveName: TeamViewerQS.dmg
2025-03-18 21:26:57 : INFO  : teamviewerqs : no blocking processes defined, using TeamViewerQS as default
2025-03-18 21:26:57 : DEBUG : teamviewerqs : Changing directory to /Users/h.lans/Documents/GitHub/Installomator/build
2025-03-18 21:26:57 : INFO  : teamviewerqs : name: TeamViewerQS, appName: TeamViewerQS.app
2025-03-18 21:26:57 : WARN  : teamviewerqs : No previous app found
2025-03-18 21:26:57 : WARN  : teamviewerqs : could not find TeamViewerQS.app
2025-03-18 21:26:57 : INFO  : teamviewerqs : appversion:
2025-03-18 21:26:57 : INFO  : teamviewerqs : Latest version of TeamViewerQS is 15.63.4
2025-03-18 21:26:57 : REQ   : teamviewerqs : Downloading https://download.teamviewer.com/download/TeamViewerQS.dmg to TeamViewerQS.dmg
2025-03-18 21:26:57 : DEBUG : teamviewerqs : No Dialog connection, just download
2025-03-18 21:27:03 : INFO  : teamviewerqs : Downloaded TeamViewerQS.dmg – Type is  zlib compressed data – SHA is 256c3a0b5beae3eaff99781160914c9ae75e6b2e – Size is 60076 kB
2025-03-18 21:27:03 : DEBUG : teamviewerqs : DEBUG mode 1, not checking for blocking processes
2025-03-18 21:27:03 : REQ   : teamviewerqs : Installing TeamViewerQS
2025-03-18 21:27:03 : INFO  : teamviewerqs : Mounting /Users/h.lans/Documents/GitHub/Installomator/build/TeamViewerQS.dmg
2025-03-18 21:27:06 : DEBUG : teamviewerqs : Debugging enabled, dmgmount output was:
Checksumming Protective Master Boot Record (MBR : 0)…
Protective Master Boot Record (MBR :: verified CRC32 $BC149447
Checksumming GPT Header (Primary GPT Header : 1)…
GPT Header (Primary GPT Header : 1): verified CRC32 $0D853B92
Checksumming GPT Partition Data (Primary GPT Table : 2)…
GPT Partition Data (Primary GPT Tabl: verified CRC32 $F093731F
Checksumming  (Apple_Free : 3)…
(Apple_Free : 3): verified CRC32 $00000000
Checksumming disk image (Apple_HFS : 4)…
disk image (Apple_HFS : 4): verified CRC32 $4C7F6BEB
Checksumming  (Apple_Free : 5)…
(Apple_Free : 5): verified CRC32 $00000000
Checksumming GPT Partition Data (Backup GPT Table : 6)…
GPT Partition Data (Backup GPT Table: verified CRC32 $F093731F
Checksumming GPT Header (Backup GPT Header : 7)…
GPT Header (Backup GPT Header : 7): verified CRC32 $FC3D163A
verified CRC32 $BD6994C0
/dev/disk4          	GUID_partition_scheme
/dev/disk4s1        	Apple_HFS                      	/Volumes/TeamViewerQS

2025-03-18 21:27:06 : INFO  : teamviewerqs : Mounted: /Volumes/TeamViewerQS
2025-03-18 21:27:06 : INFO  : teamviewerqs : Verifying: /Volumes/TeamViewerQS/TeamViewerQS.app
2025-03-18 21:27:06 : DEBUG : teamviewerqs : App size: 173M	/Volumes/TeamViewerQS/TeamViewerQS.app
2025-03-18 21:27:08 : DEBUG : teamviewerqs : Debugging enabled, App Verification output was:
/Volumes/TeamViewerQS/TeamViewerQS.app: accepted
source=Notarized Developer ID
origin=Developer ID Application: TeamViewer Germany GmbH (H7UGFBUGV6)

2025-03-18 21:27:08 : INFO  : teamviewerqs : Team ID matching: H7UGFBUGV6 (expected: H7UGFBUGV6 )
2025-03-18 21:27:08 : INFO  : teamviewerqs : Installing TeamViewerQS version 15.63.4 on versionKey CFBundleShortVersionString.
2025-03-18 21:27:08 : INFO  : teamviewerqs : App has LSMinimumSystemVersion: 12.5.1
2025-03-18 21:27:08 : DEBUG : teamviewerqs : DEBUG mode 1 enabled, skipping remove, copy and chown steps
2025-03-18 21:27:08 : INFO  : teamviewerqs : Finishing...
2025-03-18 21:27:11 : INFO  : teamviewerqs : name: TeamViewerQS, appName: TeamViewerQS.app
2025-03-18 21:27:11 : WARN  : teamviewerqs : No previous app found
2025-03-18 21:27:11 : WARN  : teamviewerqs : could not find TeamViewerQS.app
2025-03-18 21:27:11 : REQ   : teamviewerqs : Installed TeamViewerQS, version 15.63.4
2025-03-18 21:27:11 : INFO  : teamviewerqs : notifying
2025-03-18 21:27:11 : DEBUG : teamviewerqs : Unmounting /Volumes/TeamViewerQS
2025-03-18 21:27:11 : DEBUG : teamviewerqs : Debugging enabled, Unmounting output was:
"disk4" ejected.
2025-03-18 21:27:11 : DEBUG : teamviewerqs : DEBUG mode 1, not reopening anything
2025-03-18 21:27:11 : REQ   : teamviewerqs : All done!
2025-03-18 21:27:11 : REQ   : teamviewerqs : ################## End Installomator, exit code 0
````

bartreardon commented:
🤖 Validation robot 🤖
File fragments/labels/teamviewerqs.sh
** Label info:
├ ✅ correct line ending
├ ✅ Name: TeamViewerQS
├ ✅ Type: dmg
├ ✅ Expected Team: H7UGFBUGV6
├ ⚠️  App New Version: Missing
└ Download URL: https://download.teamviewer.com/download/TeamViewerQS.dmg
  └ ❌ URL is not reachable - error 0
  ├ Check [RFC status codes](https://www.rfc-editor.org/rfc/rfc9110.html#name-status-codes) for more details
** WARNING: Some checks failed
✅ 4 checks passed
❌ 1 checks failed
❌ Please review [Contributing to Installomator](https://github.com/Installomator/Installomator/wiki/Contributing-to-Installomator) and update your pull request.
**This is an automated check and response and does not cover all edge cases.**
Failed checks just mean that some automations could not complete to validation. This PR may still be validated after manual review
****

uurazzle committed:
Remove credit comment from teamviewerqs.sh

Removed credit comment for Søren Theilgaard.

uurazzle reviewed (dismissed):
Hi @hwlans,

Thanks for the updates on this Installomator label. I am approving this PR.

gilburns reviewed (changes requested):
I'm not seeing the appNewVersion variable populate with a value. I think the webpage that is reference there may have changed since you submitted this PR

uurazzle reviewed (commented):
Hi @hwlans,

Thanks for the PR.

The current `appNewVersion` implementation relies on scraping the TeamViewer website, but that page no longer exposes a reliable version string, so the pipeline can return empty.

I also tested the TeamViewer XML update feed approach, but it does not return any usable version data for QuickSupport, so it’s not a viable option.

While it’s possible to reference the Homebrew cask feed for version discovery, it only provides a major version (e.g. 15), not the full version (e.g. 15.53.6). Because Installomator compares `appNewVersion` with the installed app’s full version, using a major-only value is not ideal and can lead to inaccurate update detection.

Given that we don’t have a reliable vendor endpoint that returns the full version string remotely, it’s better in this case to avoid setting appNewVersion altogether rather than use an imprecise value.

Here’s the simplified label:

```
teamviewerqs)
    name="TeamViewerQS"
    type="dmg"
    downloadURL="https://download.teamviewer.com/download/TeamViewerQS.dmg"
    expectedTeamID="H7UGFBUGV6"
    ;;
```

This avoids fragile scraping and incorrect version comparisons, while still ensuring the latest version is downloaded each time.

If a reliable full-version source becomes available later, we can revisit adding `appNewVersion`.

hwlans committed:
Update teamviewerqs.sh

Updated `appNewVersion`

````
./assemble.sh teamviewerqs
2026-05-09 23:31:46 : INFO  : teamviewerqs : Total items in argumentsArray: 0
2026-05-09 23:31:46 : INFO  : teamviewerqs : argumentsArray:
2026-05-09 23:31:47 : REQ   : teamviewerqs : ################## Start Installomator v. 10.8beta, date 2026-05-09
2026-05-09 23:31:47 : INFO  : teamviewerqs : ################## Version: 10.8beta
2026-05-09 23:31:47 : INFO  : teamviewerqs : ################## Date: 2026-05-09
2026-05-09 23:31:47 : INFO  : teamviewerqs : ################## teamviewerqs
2026-05-09 23:31:47 : DEBUG : teamviewerqs : DEBUG mode 1 enabled.
2026-05-09 23:31:47 : INFO  : teamviewerqs : Reading arguments again:
2026-05-09 23:31:47 : DEBUG : teamviewerqs : name=TeamViewerQS
2026-05-09 23:31:47 : DEBUG : teamviewerqs : appName=
2026-05-09 23:31:47 : DEBUG : teamviewerqs : type=dmg
2026-05-09 23:31:47 : DEBUG : teamviewerqs : archiveName=
2026-05-09 23:31:47 : DEBUG : teamviewerqs : downloadURL=https://download.teamviewer.com/download/TeamViewerQS.dmg
2026-05-09 23:31:47 : DEBUG : teamviewerqs : curlOptions=
2026-05-09 23:31:47 : DEBUG : teamviewerqs : appNewVersion=15.76.6
2026-05-09 23:31:47 : DEBUG : teamviewerqs : appCustomVersion function: Not defined
2026-05-09 23:31:47 : DEBUG : teamviewerqs : versionKey=CFBundleShortVersionString
2026-05-09 23:31:47 : DEBUG : teamviewerqs : packageID=
2026-05-09 23:31:47 : DEBUG : teamviewerqs : pkgName=
2026-05-09 23:31:47 : DEBUG : teamviewerqs : choiceChangesXML=
2026-05-09 23:31:47 : DEBUG : teamviewerqs : expectedTeamID=H7UGFBUGV6
2026-05-09 23:31:47 : DEBUG : teamviewerqs : blockingProcesses=
2026-05-09 23:31:47 : DEBUG : teamviewerqs : installerTool=
2026-05-09 23:31:47 : DEBUG : teamviewerqs : CLIInstaller=
2026-05-09 23:31:47 : DEBUG : teamviewerqs : CLIArguments=
2026-05-09 23:31:47 : DEBUG : teamviewerqs : updateTool=
2026-05-09 23:31:47 : DEBUG : teamviewerqs : updateToolArguments=
2026-05-09 23:31:47 : DEBUG : teamviewerqs : updateToolRunAsCurrentUser=
2026-05-09 23:31:47 : INFO  : teamviewerqs : BLOCKING_PROCESS_ACTION=tell_user
2026-05-09 23:31:47 : INFO  : teamviewerqs : NOTIFY=success
2026-05-09 23:31:47 : INFO  : teamviewerqs : LOGGING=DEBUG
2026-05-09 23:31:47 : INFO  : teamviewerqs : LOGO=/System/Applications/App Store.app/Contents/Resources/AppIcon.icns
2026-05-09 23:31:47 : INFO  : teamviewerqs : Label type: dmg
2026-05-09 23:31:47 : INFO  : teamviewerqs : archiveName: TeamViewerQS.dmg
2026-05-09 23:31:47 : INFO  : teamviewerqs : no blocking processes defined, using TeamViewerQS as default
2026-05-09 23:31:47 : DEBUG : teamviewerqs : Changing directory to /Users/h.lans/Developer/GitHub/Installomator/build
2026-05-09 23:31:47 : INFO  : teamviewerqs : name: TeamViewerQS, appName: TeamViewerQS.app
2026-05-09 23:31:47 : WARN  : teamviewerqs : No previous app found
2026-05-09 23:31:47 : WARN  : teamviewerqs : could not find TeamViewerQS.app
2026-05-09 23:31:47 : INFO  : teamviewerqs : appversion:
2026-05-09 23:31:47 : INFO  : teamviewerqs : Latest version of TeamViewerQS is 15.76.6
2026-05-09 23:31:47 : REQ   : teamviewerqs : Downloading https://download.teamviewer.com/download/TeamViewerQS.dmg to TeamViewerQS.dmg
2026-05-09 23:31:47 : DEBUG : teamviewerqs : No Dialog connection, just download
2026-05-09 23:31:51 : INFO  : teamviewerqs : Downloaded TeamViewerQS.dmg – Type is  zlib compressed data – SHA is 4e3e4c85628798bdd36557409d3003978cbfe668 – Size is 82448 kB
2026-05-09 23:31:51 : DEBUG : teamviewerqs : DEBUG mode 1, not checking for blocking processes
2026-05-09 23:31:51 : REQ   : teamviewerqs : Installing TeamViewerQS
2026-05-09 23:31:52 : INFO  : teamviewerqs : Mounting /Users/h.lans/Developer/GitHub/Installomator/build/TeamViewerQS.dmg
2026-05-09 23:31:55 : DEBUG : teamviewerqs : Debugging enabled, dmgmount output was:
Checksumming Protective Master Boot Record (MBR : 0)…
Protective Master Boot Record (MBR :: verified CRC32 $C534DE1B
Checksumming GPT Header (Primary GPT Header : 1)…
GPT Header (Primary GPT Header : 1): verified CRC32 $56DE9D8E
Checksumming GPT Partition Data (Primary GPT Table : 2)…
GPT Partition Data (Primary GPT Tabl: verified CRC32 $D194FF85
Checksumming  (Apple_Free : 3)…
(Apple_Free : 3): verified CRC32 $00000000
Checksumming disk image (Apple_HFS : 4)…
disk image (Apple_HFS : 4): verified CRC32 $B27A5EC8
Checksumming  (Apple_Free : 5)…
(Apple_Free : 5): verified CRC32 $00000000
Checksumming GPT Partition Data (Backup GPT Table : 6)…
GPT Partition Data (Backup GPT Table: verified CRC32 $D194FF85
Checksumming GPT Header (Backup GPT Header : 7)…
GPT Header (Backup GPT Header : 7): verified CRC32 $6D6BCE2D
verified CRC32 $3DDC905A
/dev/disk4          	GUID_partition_scheme
/dev/disk4s1        	Apple_HFS                      	/Volumes/TeamViewerQS

2026-05-09 23:31:55 : INFO  : teamviewerqs : Mounted: /Volumes/TeamViewerQS
2026-05-09 23:31:55 : INFO  : teamviewerqs : Verifying: /Volumes/TeamViewerQS/TeamViewerQS.app
2026-05-09 23:31:55 : DEBUG : teamviewerqs : App size: 215M	/Volumes/TeamViewerQS/TeamViewerQS.app
2026-05-09 23:31:57 : DEBUG : teamviewerqs : Debugging enabled, App Verification output was:
/Volumes/TeamViewerQS/TeamViewerQS.app: accepted
source=Notarized Developer ID
origin=Developer ID Application: TeamViewer Germany GmbH (H7UGFBUGV6)

2026-05-09 23:31:57 : INFO  : teamviewerqs : Team ID matching: H7UGFBUGV6 (expected: H7UGFBUGV6 )
2026-05-09 23:31:57 : INFO  : teamviewerqs : Installing TeamViewerQS version 15.76.6 on versionKey CFBundleShortVersionString.
2026-05-09 23:31:57 : INFO  : teamviewerqs : App has LSMinimumSystemVersion: 13.5
2026-05-09 23:31:57 : DEBUG : teamviewerqs : DEBUG mode 1 enabled, skipping remove, copy and chown steps
2026-05-09 23:31:57 : INFO  : teamviewerqs : Finishing...
2026-05-09 23:32:00 : INFO  : teamviewerqs : name: TeamViewerQS, appName: TeamViewerQS.app
2026-05-09 23:32:00 : WARN  : teamviewerqs : No previous app found
2026-05-09 23:32:00 : WARN  : teamviewerqs : could not find TeamViewerQS.app
2026-05-09 23:32:00 : REQ   : teamviewerqs : Installed TeamViewerQS, version 15.76.6
2026-05-09 23:32:00 : INFO  : teamviewerqs : notifying
2026-05-09 23:32:00 : DEBUG : teamviewerqs : Unmounting /Volumes/TeamViewerQS
2026-05-09 23:32:00 : DEBUG : teamviewerqs : Debugging enabled, Unmounting output was:
"disk4" ejected.
2026-05-09 23:32:00 : DEBUG : teamviewerqs : DEBUG mode 1, not reopening anything
2026-05-09 23:32:00 : REQ   : teamviewerqs : All done!
2026-05-09 23:32:00 : REQ   : teamviewerqs : ################## End Installomator, exit code 0
````

uurazzle reviewed (approved):
Hi @hwlans,

Thanks for contributing this Installomator label ([#2256: Update teamviewerqs](https://github.com/Installomator/Installomator/pull/2256))—everything looks good. Appreciate you taking the time to add it to the project!

gilburns reviewed (approved):
Thank you for your contribution to the project!
